src/main/java/tc/oc/pgm/listeners/PGMListener.java

```diff
@@ -303,5 +305,27 @@ public void nerfFishing(PlayerFishEvent event) {
         caught.setItemStack(new ItemStack(Material.RAW_FISH));
       }
     }
+
+
+  }
+
+  @EventHandler(priority = EventPriority.MONITOR)
+  public void dropItemsOnQuit(PlayerParticipationStopEvent event) {
+    MatchPlayer quitter = event.getPlayer();
+    Player player = Bukkit.getPlayer(quitter.getId());
```

Comment:
There must be a getter for this in `MatchPlayer` for sure.

Reply:
You mean MatchPlayer.getBukkit()? It was deprecated so i avoided it

Reply:
@KingOfSquares No, please use `MatchPlayer#getBukkit`